ahoy.js

```diff
@@ -345,27 +368,39 @@
   };
 
   ahoy.trackClicks = function () {
-    $(document).on("click", "a, button, input[type=submit]", function (e) {
-      var $target = $(e.currentTarget);
-      var properties = eventProperties(e);
-      properties.text = properties.tag == "input" ? $target.val() : $.trim($target.text().replace(/[\s\r\n]+/g, " "));
-      properties.href = $target.attr("href");
-      ahoy.track("$click", properties);
-    });
+    var elements = document.querySelectorAll("a, button, input[type=submit]");
+
+    for (var i = 0; i < elements.length; i++) {
+      elements[i].addEventListener('click', function (e) {
```

Comment:
Can we DRY up the `querySelectorAll`, loop over elements, `addEventListener` code?